tests/features/unittest_aliases_test.py

```diff
@@ -7,14 +7,17 @@
 
 
 @pytest.mark.parametrize(
-    ('s',),
+    's',
     (
         pytest.param(
             'class ExampleTests:\n'
             '    def test_something(self):\n'
             '        self.assertEqual(1, 1)\n',
             id='not a deprecated alias',
         ),
+        # https://github.com/asottile/pyupgrade/issues/749#
```

Comment:
if this link is important put it in the pytest id -- if it's not remove it -- also it contains a typo